pkgs/tools/typesetting/sile/default.nix

```diff
@@ -46,11 +44,12 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   checkPhase = ''
-    make documentation/developers.pdf documentation/sile.pdf
+    make documentation examples
```

Comment:
weird to build documentation in a checkPhase. Eventually documentation generation could be toggled via a withDoc flag.

Reply:
Should I be adding `--with-documentation` as a `./configure` flag so that `make all` includes these (or not) without needing to know to trigger the extra targets?

Reply:
Documentation can be dropped from here, `make examples` is actually not a bad way to do a simple check that everything is working. The downside is going to be that it downloads a bunch of fonts to do that. `make busted` would be a better check, but requires extra dependencies. Just checking `./sile --version` might be an acceptable check phase.